### PR TITLE
Fix destination paths.

### PR DESCRIPTION
**Pull request by steveperry-53**
Fix a few destination paths in the reference configuration file for the update-imported-docs tool.

### CODE REVIEW

File: update-imported-docs/reference.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ repos:
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_all.md
   - src: docs/admin/kubeadm_alpha_phase_certs_apiserver-etcd-client.md
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_apiserver-etcd-client.md
-  - src: docs/admin/kubeadm_alpha_phase_certs_apiserver-kubelet-client.md
+- src: docs/admin/kubeadm_alpha_phase_certs_apiserver-kubelet-client.md
```
Comment (neolit123) [inline update-imported-docs/reference.yml]: [1] given this is yaml, shouldn't the above indented with 4 spaces.

File: update-imported-docs/reference.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ repos:
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_kubeconfig.md
   - src: docs/admin/kubeadm_alpha_phase_kubeconfig_scheduler.md
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_kubeconfig_scheduler.md
-  - src: docs/admin/kubeadm_alpha_phase_kubeconfig_user.md
+- src: docs/admin/kubeadm_alpha_phase_kubeconfig_user.md
```
Comment (neolit123) [inline update-imported-docs/reference.yml]: [1]

File: update-imported-docs/reference.yml [hunk 3]
```diff
@@ -142,7 +142,7 @@ repos:
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md
   - src: docs/admin/kubeadm_config_upload_from-file.md
     dst: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_upload_from-file.md
-  - src: docs/admin/kubeadm_config_upload_from-flags.md
+- src: docs/admin/kubeadm_config_upload_from-flags.md
```
Comment (neolit123) [inline update-imported-docs/reference.yml]: [1]

**Comment by steveperry-53** [comment]
@neolit123 Thanks. I fixed the three indentation mistakes.

**Pull request merged**